pages/content/amp-dev/documentation/guides-and-tutorials/start/create/index.md

```diff
@@ -1,14 +1,18 @@
 ---
 $title: Create your first AMP page
 $order: 0
-description: 'Not sure how to get started? In this tutorial, you’ll learn how to create a basic AMP HTML page, how to stage it and validate that it’s AMP compliant, and finally ...'
+description: 'In this tutorial, we’ll introduce you to the basics of an AMP page!'
 formats:
   - websites
 numbered: 0
 $hidden: true
-author: pbakaus
-contributors:
-  - bpaduch
+author: crystalonscript
 ---
 
-Not sure how to get started? In this tutorial, you’ll learn how to create a basic AMP HTML page, how to stage it and validate that it’s AMP compliant, and finally how to get it ready for publication and distribution.
+In this tutorial, we’ll introduce you to the basics of an AMP page. Follow these steps to get started learning how AMP is unique, checking if a page is valid, and taking next steps to publishing it to the web.
+
+AMP pages that pass validation are more likely to meet [Page Experience metrics](https://amp.dev/about/page-experience/) and enable caching. If you are uninterested in AMP cache benefits, it is okay to create invalid AMP pages or use [Bento AMP](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/?format=websites).
```

Comment:
There should be a caveat here that invalid AMP pages aren't a directly supported use case in general. Bento AMP gives components that work on non-AMP pages (or invalid AMP pages), but there's no guarantee on if the rest of an invalid AMP page will work correctly.

Reply:
Thanks for flagging! I've updated the wording, lmkwyt!